packages/component-library/src/components/graphs/mt-graph/mt-graph-default-options.ts

```diff
@@ -0,0 +1,48 @@
+import type { ApexOptions } from "apexcharts";
+
+const defaultOptions: { [key: string]: ApexOptions } = {
```

Comment:
IMO we should use a factory method here instead of a singleton

```suggestion
export function getDefaultAreaOptions: ApexOptions = {
  return {
    noData: {
      [...]
    }
  };
}
```

Reply:
Better this way?
I was struggling since i wanted to be able to support different chart types in the future.